gems/quilt_rails/lib/generators/quilt_rails/quilt/react_setup/templates/tsconfig.json

```diff
@@ -2,7 +2,10 @@
   "extends": "@shopify/typescript-configs/application.json",
   "compilerOptions": {
     "baseUrl": ".",
-    "rootDir": "."
+    "rootDir": ".",
+    "paths": {
+      "*": ["app/ui/*"]
+    }
   },
-  "include": ["app/ui"]
+  "include": ["config/*.ts", "./app/ui/**/*"]
```

Comment:
Nit: either keep all paths app-relative, or start all paths with `./`.  Otherwise, it looks like there's something special about `./app/ui/**/*`.